CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Cromwell Change Log
 
+## 47 Release Notes
+
+### Retry with double memory [(#5180)](https://github.com/broadinstitute/cromwell/pull/5180)
```

Comment:
It looks like even though this is backend agnostic code, that the implementation was embedded w/ Papi for now. Thus I wouldn't get the other backend-users hopes up. Perhaps "Retry with double memory on Papi"

Reply:
I think there may be a producty question for @ruchim as to whether the more/double memory retry system should support other backends. The [relevant JIRA](https://broadworkbench.atlassian.net/browse/BA-5933) doesn't say anything about this being PAPI specific but I'm not sure what was intended.

Reply:
It's only relevant for PAPI for now, and let's make sure the release notes reflect that!

Reply:
I have added `on Papiv2` at the end